src/main/java/com/google/firebase/messaging/Duration.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.messaging;
+
+import com.google.api.client.util.Key;
+
+/**
+ * A class representing duration in an LightSettings.
+ */
+public class Duration {
+  
+  @Key("seconds")
+  private final Long seconds;
+  
+  @Key("nanos")
+  private final Integer nanos;
+
+  private Duration(Builder builder) {
```

Comment:
Again a Builder is probably overkill for this class. May be just `public Duration(long seconds, long nanos)`

Reply:
Done.